changelogs/fragments/9263-kc_authentication-api-priority.yaml

```
@@ -0,0 +1,2 @@
+security_fixes:
+  - keycloak_authentication - API calls did not properly set the `priority` during update resulting in incorrectly sorted authentication flows (https://github.com/ansible-collections/community.general/pull/9263).
```

I would explicitly mention that this is apparently a problem with Keycloak 25 or newer.
```suggestion
  - keycloak_authentication - API calls did not properly set the ``priority`` during update resulting in incorrectly sorted authentication flows. This apparently only affects Keycloak 25 or newer (https://github.com/ansible-collections/community.general/pull/9263).
```